core/domain/email_manager.py

```diff
@@ -489,6 +655,28 @@ def send_feedback_message_email(recipient_id, feedback_messages):
 
 def send_suggestion_email(
         exploration_title, exploration_id, author_id, recipient_list):
+    """Sends emails to notify of a new suggestion.
+
+    Each recipient will only be emailed if they have receiving feedback message
```

Comment:
if their email preferences allow for incoming feedback message emails.

(The last sentence can be dropped. It's an implementation detail.)